src/Acuminator/Acuminator.Utilities/Roslyn/PXFieldAttributes/AttributeInformation.cs

```diff
@@ -181,17 +193,35 @@ public BoundType GetBoundAttributeType(AttributeData attribute)
 			
 			if (containsIsDbFieldproperty)
 			{
-				var isDbPropertyAttributeArgs = attribute.NamedArguments.Where(arg => IsDBField.Equals(arg.Key, StringComparison.OrdinalIgnoreCase)).ToList();    //case insensitive check
+				var isDbPropertyAttributeArgs = attribute.NamedArguments.Where(arg => IsDBField.Equals(arg.Key, StringComparison.OrdinalIgnoreCase)).ToList();
 
-				if (isDbPropertyAttributeArgs.Count != 1)  //rare case when there are multiple different "IsDBField" considered
-					return BoundType.Unknown;
+				if (isDbPropertyAttributeArgs.Count > 0)
+				{
+					if (isDbPropertyAttributeArgs.Count != 1)  //rare case when there are multiple different "IsDBField" considered
+						return BoundType.Unknown;
 
-				if (!(isDbPropertyAttributeArgs[0].Value.Value is bool isDbPropertyAttributeArgument))
-					return BoundType.Unknown;  //if there is null or values of type other than bool then we don't know if attribute is bound
+					if (!(isDbPropertyAttributeArgs[0].Value.Value is bool isDbPropertyAttributeArgument))
+						return BoundType.Unknown;  //if there is null or values of type other than bool then we don't know if attribute is bound
 
-				return isDbPropertyAttributeArgument
-					? BoundType.DbBound
-					: BoundType.Unbound;
+					return isDbPropertyAttributeArgument
+						? BoundType.DbBound
+						: BoundType.Unbound;
+
+				}
+				else
+				{
+					ITypeSymbol typeFromRegister = TypesContainingIsDBField.Keys.FirstOrDefault(t => IsAttributeDerivedFromClass(attribute.AttributeClass, t));
+					bool? isDbFieldFromBase = typeFromRegister != null
+						? TypesContainingIsDBField[typeFromRegister]
+						: (bool?)null;
```

Comment:
Why can't you assign `null` instead of `(bool?)null`?

Reply:
@metallmonster it's because C# compiler can't determine the type of conditional expression without it. It shows compile error if you assign null.

Reply:
Ok